CreeDictionary/DatabaseManager/test_db_builder.py

```diff
@@ -49,15 +49,15 @@ def build_test_xml(multi_processing: int = 2):
     engcrk_root = ET.parse(str(engcrk_file_path)).getroot()
 
     # relevant entries in crkeng.xml file we want to determine
-    relevant_xml_lemmas: Set[str] = set()
+    relevant_xml_ls: Set[str] = set()
     # relevant entries in engcrk.xml file we want to determine
     relevant_eng_words: Set[str] = set()
 
-    xml_lemmas: Set[str] = set()
+    xml_ls: Set[str] = set()
     crkeng_entries = crkeng_root.findall(".//e")
     for element in crkeng_entries:
-        xml_lemma = extract_l_str(element)
-        xml_lemmas.add(xml_lemma)
+        xml_l = extract_l_str(element)
+        xml_ls.add(xml_l)
```

Comment:
xml_lemma -> xml_l

Reply:
I was planning to use `l` only, but some PEP warned me that `l` shouldn't be used as a variable name cuz it's a confusion